### PR TITLE
docs: update candlestick visualization

### PR DESCRIPTION
**Pull request by mdcruz**
**What is this feature?**

Updates to the candlestick docs to include:
- how to configure a candlestick (video TBA)
- supported data formats and example
- when to use a candlestick
- play shortcode for candlestick

**Why do we need this feature?**

To explain to our users how to configure a candlestick easily and when to use it.

**Who is this feature for?**

Grafana OSS and Cloud users

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md [hunk 1]
```diff
@@ -21,12 +21,53 @@ weight: 100
...
 # Candlestick
 
-The candlestick visualization allows you to visualize data that includes a number of consistent dimensions focused on price movement. The candlestick visualization includes an Open-High-Low-Close (OHLC) mode, as well as support for additional dimensions based on time series data.
-
-{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1200px" caption="Candlestick visualization" >}}
+The candlestick visualization allows you to visualize data that includes a number of consistent dimensions focused on price movements, such as stock prices. The candlestick visualization includes an [Open-High-Low-Close (OHLC) mode](#open-high-low-close), as well as support for additional dimensions based on time series data.
 
 Candlestick visualizations build upon the foundation of the [time series visualization][] and include many common configuration settings.
 
+For example, you can use a candlestick if you want to visualize at a glance how the price moved over time, whether it went up, down, or stayed the same, and how much it fluctuated:
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/candlestick/index.md:28]:
```suggestion
You can use a candlestick if you want to visualize, at a glance, how a price moved over time, whether it went up, down, or stayed the same, and how much it fluctuated:
```

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md [hunk 2]
```diff
@@ -21,12 +21,53 @@ weight: 100
...
+
+{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1065px" alt="A candlestick visualization" >}}
+
+Each candlestick is represented as a rectangle, referred to as the "candlestick body". The candlestick body displays the opening and closing prices during a time period. The green candlesticks represent when the price appreciated while the red candlesticks represent when the price depreciated. The lines sticking out the candlestick body are referred to as "wicks" or "shadows", which represent the highest and lowest prices during the time period.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/candlestick/index.md:32]:
```suggestion
Each candlestick is represented as a rectangle, referred to as the _candlestick body_. The candlestick body displays the opening and closing prices during a time period. Green candlesticks represent when the price appreciated while the red candlesticks represent when the price depreciated. The lines sticking out the candlestick body are referred to as _wicks_ or _shadows_, which represent the highest and lowest prices during the time period.
```

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md [hunk 3]
```diff
@@ -21,12 +21,53 @@ weight: 100
...
+
+- Monitor and identify trends in price movements of specific assets such as stocks, currencies, or commodities.
+- Analyze any volatility in the stock market.
+- Provide data analysis to help with trading decisions.
+
+## Configure a candlestick
+
+<!-- video TBA here -->
+
+{{< docs/play title="Candlestick Visualizations in Grafana" url="https://play.grafana.org/d/candlestick/candlestick" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/candlestick/index.md:44]:
```suggestion
{{< docs/play title="Candlestick" url="https://play.grafana.org/d/candlestick/candlestick" >}}
```

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md [hunk 4]
```diff
@@ -21,12 +21,53 @@ weight: 100
...
+
+## Supported data formats
+
+The candlestick visualization works best if you have the price movements of the asset that you are interested in tracking. The data must include:
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/candlestick/index.md:48]:
```suggestion
The candlestick visualization works best with price movement data for an asset. The data must include:
```

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md [hunk 5]
```diff
@@ -21,12 +21,53 @@ weight: 100
...
+
+- **Timestamps** - The time period for when each price movement occurred.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/candlestick/index.md:50]: This is a bit unclear. My understanding is each individual price movement has a specific timestamp.
```suggestion
- **Timestamps** - The time at which each price movement occurred.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for starting this! I've made some suggestions and left a question/suggested edit on line 50.

File: docs/sources/panels-visualizations/visualizations/candlestick/index.md (hunk 5 above)
Comment (mdcruz) [reply docs/sources/panels-visualizations/visualizations/candlestick/index.md:50]: Yes, your understanding is correct! Thank you for correcting it :)

**Comment by mdcruz** [comment]
> Thanks for starting this! I've made some suggestions and left a question/suggested edit on line 50.

Thanks, Isabel! I've committed your suggestions and also added the Candlestick video, which will go live on Monday, the 22nd of April.

**Review by imatwawana — APPROVED** [review]

**Pull request merged**